cmd/operator-sdk/generate/packagemanifests/cmd.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package packagemanifests
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	kbutil "github.com/operator-framework/operator-sdk/internal/util/kubebuilder"
+	"github.com/operator-framework/operator-sdk/internal/util/projutil"
+)
+
+//nolint:maligned
+type packagemanifestsCmd struct {
+	// Options to turn on different parts of packaging.
+	kustomize bool
+	manifests bool
+
+	// Common options.
+	operatorName string
+	version      string
+	fromVersion  string
+	inputDir     string
+	outputDir    string
+	deployDir    string
+	apisDir      string
+	crdsDir      string
+	updateCRDs   bool
+	stdout       bool
+	quiet        bool
+
+	// Interactive options.
+	interactiveLevel projutil.InteractiveLevel
+	interactive      bool
+
+	// Package manifest options.
+	channelName      string
+	isDefaultChannel bool
+}
+
+// NewCmd returns the 'packagemanifests' command configured for the new project layout.
+func NewCmd() *cobra.Command {
+	c := &packagemanifestsCmd{}
+
+	cmd := &cobra.Command{
+		Use:   "packagemanifests",
+		Short: "Generates a package manifests format",
```

Comment:
Can we add an example for `generate packagemanifests` here like we did for `generate bundle`?

Especially since the first time workflow requires multiple invocations of the command:
```console
$ operator-sdk generate packagemanifest -q --kustomize

$ kustomize build config/packages | operator-sdk generate packagemanifests -q --channel beta --version 0.0.1
```

Reply:
Ah whoops, I got tripped up on the workflow :) Going off the enhancement since there wasn't an example:
https://github.com/operator-framework/enhancements/blob/master/enhancements/sdk-generate-commands.md#generate-packagemanifests

That's probably outdated as it states:
`kustomize build config/packages | operator-sdk generate packagemanifests`
instead of 
`kustomize build config/default | operator-sdk generate packagemanifests`